modules/integration/src/test/scala/scala/cli/integration/SipScalaTests.scala

```diff
@@ -515,6 +515,39 @@ class SipScalaTests extends ScalaCliSuite with SbtTestHelper with MillTestHelper
     }
   }
 
+  test("consecutive -Wconf:* flags are not ignored") {
+    val sv                 = "3.5.2"
+    val sourceFileName     = "example.scala"
+    val warningConfOptions = Seq("-Wconf:cat=deprecation:e", "-Wconf:any:s")
```

Comment:
How about adding some noise here by adding an option which shouldn't change the compiler's behaviour in this context, just to make sure that it's not only the last or the first option that is passed to the compiler, while the rest would be ignored?

Reply:
...like?

Reply:
I think this case is sufficient, as both options affect the output in tandem, so it won't work correctly unless both are applied... And `-Wconf` is quite tweaky, so I'd rather not make it more complex, to make referring to this test later easier...

Reply:
I mean e.g. currently (with scala-cli 1.5.1) the following code raises an error
```scala
//> using scala 3.5.2
//> using options -Wconf:cat=deprecation:e -Wconf:any:s
object WConfExample extends App {
  @deprecated("This method will be removed", "1.0.0")
  def oldMethod(): Unit = println("This is an old method.")
  oldMethod()
}
```
but this has just a warning
```scala
//> using scala 3.5.2
//> using options -Wconf:cat=unchecked:s -Wconf:cat=deprecation:e -Wconf:any:s
object WConfExample extends App {
  @deprecated("This method will be removed", "1.0.0")
  def oldMethod(): Unit = println("This is an old method.")
  oldMethod()
}
```
In this test it's not really clear if the given behaviour is a result of reshuffling options or discarding one/some of them